cellprofiler/modules/measuretexture.py

```diff
@@ -84,6 +87,39 @@
       relationship between specific intensity values. It is a complementary
       value to InfoMeas1 and is on a different scale.
 
+**Note**: each of the above measurements are computed for different 
+'directions' in the image, specified by a series of correspondence vectors. 
+These are indicated in the results table in the *scale* column as n_00, n_01,
+n_02... for each scale *n*. In 2D, the directions and correspondence vectors *(y, x)* 
+for each measurement are given below:
+
+- _00 = horizontal, 0 degrees   (0, 1)
+- _01 = diagonal, 135 degrees   (1, 1)
+- _02 = vertical, 90 degrees   (1, 0)
+- _03 = diagonal, 45 degrees   (1, -1)
+
+When analyzing 3D images, there are 13 correspondence vectors *(y, x, z)*:
+
+- (1, 0, 0)
+- (1, 1, 0)
+- (0, 1, 0)
+- (1,-1, 0)
+- (0, 0, 1)
+- (1, 0, 1)
+- (0, 1, 1)
+- (1, 1, 1)
+- (1,-1, 1)
+- (1, 0,-1)
+- (0, 1,-1)
+- (1, 1,-1)
+- (1,-1,-1)
+
+In this case, an image makes understanding their directions easier. 
+Imagine the origin (0, 0, 0) is at the upper left corner of the first image
+in your z-stack.
+
+|MT_image0|
```

Comment:
How tricky is it to edit the image? I feel like the angle it's at right now makes it a little challenging for me personally to interpret, it might be nice to try a couple rotations but I don't know how doable that is. (Also, should we credit how we made the image?)

Reply:
I've added a credit in for the website I used to make it (GeoGebra) We can rotate! If helpful, I could include a few views of the same image: 
![image](https://user-images.githubusercontent.com/28116530/153264917-18f6e77b-87c7-498e-9c6a-417b70216437.png)
![image](https://user-images.githubusercontent.com/28116530/153264960-36858988-3ad9-4cb1-8279-495db7610359.png)
We can also show the XY plane and/or change up the colorcoding, though I personally get a 'too much' vibe from this: 
![image](https://user-images.githubusercontent.com/28116530/153265820-7f701c5f-e695-4e7f-ad4a-83277910b559.png)